lib/src/main/java/io/ably/lib/realtime/AblyRealtime.java

```diff
@@ -71,6 +71,13 @@ public void onAuthUpdated() {
 	}
 
 	/**
+	 * Authentication error occurred
+	 */
+	public void onAuthError(ErrorInfo errorInfo) {
```

Comment:
Is there not a convention at least to indicate that this is not a method people should be calling in the documentation at least then?  Seems horrible that we are exposing all these methods to users, especially in langs like Java where most people use IDEs with intellisense.